specification/loadtestservice/resource-manager/Microsoft.LoadTestService/preview/2021-11-01-preview/examples/LoadTests_CreateOrUpdate.json

```diff
@@ -0,0 +1,43 @@
+{
+  "parameters": {
+    "api-version": "2021-11-01-preview",
+    "subscriptionId": "00000000-0000-0000-0000-000000000000",
+    "resourceGroupName": "dummyrg",
+    "loadTestName": "myLoadTest",
+    "LoadTestResource": {
+      "location": "westus",
+      "tags": {
+        "Team": "Dev Exp"
+      },
+      "properties": {
+        "description": "This is new load test resource"
+      }
+    }
```

Comment:
This looks like description and name is the only arguments that is required from the customer to create whis resource. Can you please expand on how would the customer interact with this resource ? Like can they do anything on the resource from the control plane other than CRUD ?


The reason we ask this is to please note that people use ARM templates to configure azure resources all the time. and this resource seems like  it is not configurable at all when using an ARM template.

Reply:
@raosuhas , the customer can only perform CRUD operations on the resource as most of our business logic lies in data-plane (we'll be submitting data-plane apis too for review soon). We just need name, description and location from the customer to create the resource.